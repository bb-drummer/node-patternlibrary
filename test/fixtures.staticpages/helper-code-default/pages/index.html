{{#code}}
<section class="galaxy">
    <div class="planet">
        <h1 id="earth">
            some markup...
        </h1>
    </div>
    <span id="sun">
        more markup...
    </span>
</section>
{{/code}}
