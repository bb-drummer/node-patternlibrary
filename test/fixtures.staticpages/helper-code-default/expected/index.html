<html>
  <body>
    <div class="code-example"><pre><code class="html"><span class="hljs-tag">&lt;<span class="hljs-name">section</span> <span class="hljs-attr">class</span>=<span class="hljs-string">"galaxy"</span>&gt;</span>
        <span class="hljs-tag">&lt;<span class="hljs-name">div</span> <span class="hljs-attr">class</span>=<span class="hljs-string">"planet"</span>&gt;</span>
            <span class="hljs-tag">&lt;<span class="hljs-name">h1</span> <span class="hljs-attr">id</span>=<span class="hljs-string">"earth"</span>&gt;</span>
                some markup...
            <span class="hljs-tag">&lt;/<span class="hljs-name">h1</span>&gt;</span>
        <span class="hljs-tag">&lt;/<span class="hljs-name">div</span>&gt;</span>
        <span class="hljs-tag">&lt;<span class="hljs-name">span</span> <span class="hljs-attr">id</span>=<span class="hljs-string">"sun"</span>&gt;</span>
            more markup...
        <span class="hljs-tag">&lt;/<span class="hljs-name">span</span>&gt;</span>
    <span class="hljs-tag">&lt;/<span class="hljs-name">section</span>&gt;</span>
    </code></pre></div>  </body>
</html>
